extensions/integration-tests/src/stressutils.ts

```diff
@@ -0,0 +1,156 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/*
+	This contains all the definitions for Stress decorators and the utility functions and definitions thereof
+*/
+import 'mocha';
+import { AssertionError } from 'assert';
+import { getSuiteType, SuiteType } from './fmkUtils';
+import assert = require('assert');
+
+export interface StressOptions {
+	runtime?: number;
+	dop?: number;
+	iterations?: number;
+	passThreshold?: number;
+}
+
+const DefaultStressOptions: StressOptions = { runtime: 120, dop: 4, iterations: 10, passThreshold: 0.95 };
+
+// This simulates a sleep where the thread is suspended without spinning for a given number of milliseconds before resuming
+//
+export function sleep(ms: number) {
+	return (async () => {
+		return await new Promise((undefined) => setTimeout(undefined, ms));
+	})();
+}
+
+// This is just a synonym for sleep(0). This has the affect of yielding to other operations.
+//
+export function bear() {
+	return sleep(0);
+}
+
+class Stress {
+	// number of iterations.
+	iterations?: number;
+	// seconds
+	runtime?: number;
+	// degree of parallelism
+	dop?: number;
+	// threshold of individual test passes to declare the stress test passed. This is a fraction within 1.
+	passThreshold?: number;
+
+	constructor({ runtime = parseInt(process.env.StressRunTime), dop = parseInt(process.env.StressDop), iterations = parseInt(process.env.StressIterations), passThreshold = parseFloat(process.env.StressPassThreshold) }: StressOptions = DefaultStressOptions) {
+		//console.log (`runtime=${runtime}, dop=${dop}, iterations=${iterations}, passThreshold=${passThreshold}`);
```

Comment:
dead-code

Reply:
I am going to replace these with configurable logging scheme so do not want to lose these logs.